test-monitor.cpp

```diff
@@ -2,7 +2,43 @@
 
 #include "gtest/gtest.h"
 
-TEST(Monitor, NotOkWhenAnyVitalIsOffRange) {
-  ASSERT_FALSE(vitalsOk(99, 102, 70));
-  ASSERT_TRUE(vitalsOk(98.1, 70, 98));
+std::vector<VitalValue> vitalsAllOKData = {
+    {Temperature, 96},
+    {PulseRate, 70},
+    {SPO2, 95}
+};
+
+TEST(Monitor, AllVitalsOK) {
+    ASSERT_TRUE(vitalsOk(vitalsAllOKData));
+}
+
+TEST(Monitor, HighTemperature) {
+    std::vector<VitalValue> highTempData = vitalsAllOKData;
+    highTempData[Temperature].value = 103;

```

Comment:
Good way to alter only the vital to be made not-ok, instead of declaring every vital in every test case